### ISSUE TITLE
load a DNN without knowing what is inside

### ISSUE THREAD

**Issue by chivalry123** [tflearn/tflearn#399]
Suppose I want to load a DNN where I do not know what is inside ... is it possible?

for pybrain this is like:

`            # self._net = NetworkReader.readFrom(self._load_nn_location)
`
I tried to do the same with:

```
            self._net = tflearn.DNN()
            self._net.load(self._load_nn_location)

```

but i it would not work, right?

**Comment by amirbar** [comment]
I don't think it would.. you basically have to first define your variables , i.e in your case the network variables, and only then load the weights.

**Comment by aymericdamien** [comment]
Yes, it is not possible, you need to build your network first.
